apis/v1alpha2/config.go

```diff
@@ -69,27 +73,45 @@ func (c *AnyConfig) MarshalJSON() ([]byte, error) {
 	return json.Marshal(c.Object)
 }
 
+// UnmarshalYAML implements an alternative parser this field.
+func (c *AnyConfig) UnmarshalYAML(value *yaml.Node) error {
+	vals := map[string]interface{}{}
+	if err := value.Decode(&vals); err != nil {
+		return err
+	}
+	c.Object = vals
+	return nil
+}
+
+// MarshalYAML specifies how to convert this object into YAML.
+func (c *AnyConfig) MarshalYAML() (interface{}, error) {
```

Comment:
Does the PR fixe the issue without defining `UnmarshalYAML` and `MarshalYAML`

Reply:
What do you mean?

Reply:
Do we need to have defined these two methods? 

If we don't define them would the test added in this PR fail?

Reply:
Without the methods, the test would fail. Because the default unmarshal/marshal methods take the `Object` member into account.